services/api/mw2pdf/src/classes/Pdf.ts

```diff
@@ -1,36 +1,107 @@
 import fs from 'fs'
 import { getDocument } from 'pdfjs-dist/es5/build/pdf'
 
+
+/**
+ * A data structure to hold parameters that control the construction
+ * of a PDF.
+ */
+export class PdfConstructorOptions {
+  title: string;
+  filename: string;
+
+  /**
+   *
+   * Create a [[PdfConstructorOptions]] data structure.
+   *
+   * @param title The title of the PDF.
+   * @param filename The filename where the PDF is stored on disk.
+   * This is a fully-qualified path.
+   */
+  constructor(title: string, filename: string) {
+    this.title = title;
+    this.filename = filename;
+  }
+}
+
+/**
+ * A data structure to hold information about the size of a PDF
+ */
+export class PdfSize {
```

Comment:
Once we add linting I know we're gonna get slapped with the "too many classes in one file" warning.

That said I think most of these are actually just [`Interfaces`](https://www.typescriptlang.org/docs/handbook/interfaces.html) instead of classes, so we should be able to just make that swap and won't violate the one-class-to-a-file goal.